setup.py

```diff
@@ -181,8 +75,9 @@ def read_requirements(path):
     author_email=ABOUT['__email__'],
     description=ABOUT['__summary__'],
     license=ABOUT['__license__'],
-    long_description_content_type="text/markdown",
-    long_description_markdown_filename='README.md',
-    keywords="nucypher, proxy re-encryption",
-    classifiers=PYPI_CLASSIFIERS
+    keywords="nucypher, threshold, taco, "
+    "threshold cryptography, "
+    "distributed key generation, "
+    "proxy re-encryption",
```

Comment:
should we get rid of "proxy re-encryption" and use "threshold access contract" / "taco"?

Reply:
✔️ thinned it out a bit